modules/search/partials/vector-search-field-descriptions.adoc

```diff
@@ -31,6 +31,15 @@ Smaller euclidean distances mean that the values of each coordinate in the vecto
 +
 It's best to use *l2_norm* similarity when your embeddings contain information about the count or measure of specific things, and your embedding model uses the same similarity metric. 
 
+* *cosine*: Calculated by adding the result of multiplying a vector's components, or the product of the magnitudes of the vectors and the cosine of the angle between them.
+This metric is not affected by the size of the vectors being measured.
++
+Use *cosine* similarity to get the best results with an embedding model that uses cosine similarity.
+Cosine similarity works well for semantic search, document classification, and recommendation systems.
++
+The Search Service will normalize any vectors before indexing when using cosine similarity.
```

Comment:
Worth mentioning that - we will also normalize the query vector prior to executing the nearest neighbor search if the field it is associated with is using the `cosine` similarity distance metric.